dbt/include/snowflake/macros/adapters.sql

```diff
@@ -193,6 +193,11 @@
 
 
 {% macro set_query_tag() -%}
+    {{ return(adapter.dispatch('set_query_tag', 'dbt')()) }}
+{% endmacro %}
+
+
+{% macro default__set_query_tag() -%}
```

Comment:
Nit: I feel like these should be `snowflake__`, rather than `default__`.

This makes no functional difference today, but in a world where another database comes along that _also_ supports "query tags," and we wanted to move the dispatched `set_query_tag` macro into `dbt-core` — the macro here would want to be `snowflake__set_query_tag()`, and the macro in `otherdb` would want to be `otherdb__set_query_tag()`.

Do you buy it?

Reply:
SGTM. Resolved in [bcf9695](https://github.com/dbt-labs/dbt-snowflake/pull/132/commits/bcf96953cd552b74d506647154f7fa68cbd3a1bf)